.github/workflows/continuous-integration.yml

```diff
@@ -18,10 +18,16 @@ jobs:
   # Make sure eslint passes
   eslint:
     uses: Seneca-CDOT/telescope/.github/workflows/eslint-ci.yml@master
+    secrets:
+      github-token: ${{ secrets.GITHUB_TOKEN }}
```

Comment:
I think we can't pass this in (it's a synthesized token, not a real one), see https://github.community/t/how-to-pass-github-token-to-reusable-workflows/204905/6.  What if you use `{{ github.token }}` below instead and get rid of this.